docs/guides/tutorials/rest_apis_with_fastapi.rst

```diff
@@ -0,0 +1,535 @@
+.. _ref_guide_rest_apis_with_fastapi:
+
+===============================
+REST APIs with EdgeDB & FastAPI
+===============================
+
+EdgeDB can help you quickly build REST APIs in Python without getting into the rigmarole of using ORM libraries to handle your data effectively. Here, we'll be using `FastAPI <https://fastapi.tiangolo.com/>`_ to expose the API endpoints and EdgeDB to store the content.
+
+We'll build a simple event management system where you'll be able to fetch, create, update, and delete *event hosts* and *events* via RESTful API endpoints.
+
+Prerequisites
+=============
+
+Before we start, make sure you've :ref:`installed <ref_admin_install>` EdgeDB and EdgeDB-CLI. In this tutorial, we'll use Python 3.10 and take advantage of the asynchronous I/O paradigm to communicate with the database more efficiently. A working version of this tutorial can be found `here <https://github.com/edgedb/edgedb-examples/tree/main/fastapi-crud>`_ on GitHub.
+
+
+Install the dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To follow along, clone the repository and head over to the ``fastapi-crud`` directory. Create a Python 3.10 virtual environment, activate it, and install the dependencies with this command:
+
+.. code-block:: bash
+    pip install edgedb fastapi httpx[cli] uvicorn
+
+
+Initialize the database
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Now, let's create a new EdgeDB instance for this project. From the root directory, run:
+
+.. code-block:: bash
+    edgedb project init
```

Comment:
I think you're supposed to have an empty line between ReST directives (`. code-block:: bash` in this case) and its body (`edgedb project init` in this case), like so:

```suggestion
.. code-block:: bash

    edgedb project init
```

Reply:
I was able to build the site locally. These should be resolved.